tensorboard/webapp/analytics/analytics_module.ts

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {NgModule} from '@angular/core';
+import {EffectsModule} from '@ngrx/effects';
+
+import {AnalyticsEffects} from './effects';
+import {AnalyticsLogger, GoogleAnalyticsLogger} from './analytics_logger';
+
+@NgModule({
+  imports: [EffectsModule.forFeature([AnalyticsEffects])],
+  providers: [{provide: AnalyticsLogger, useClass: GoogleAnalyticsLogger}],
```

Comment:
Interesting. I thought the class being provided, `AnalyticsLogger`, require to be an `Injectable`. Interesting. 🤔